README.md

```diff
@@ -7,9 +7,10 @@
 [![](https://images.microbadger.com/badges/image/gitea/gitea.svg)](http://microbadger.com/images/gitea/gitea "Get your own image badge on microbadger.com")
 [![Join the chat at https://gitter.im/go-gitea/gitea](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/go-gitea/gitea?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![](https://github.com/go-gitea/gitea/blob/master/public/img/gogs-large-resize.png?raw=true)
+![](https://github.com/go-gitea/gitea/blob/master/public/img/gitea-large-resize.png?raw=true)
 
-##### Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases) for binary versions or submit a task on [alpha stage automated binary building system](https://build.gogs.io/))
+##### Status
+**Current version: 1.0** (see [Releases](https://github.com/go-gitea/gitea/releases)) 
```

Comment:
I think versions have 3 numbers, so would be 1.0.0 (but there's no such release made yet).
I agree on dropping this paragraph completely and just link to the releases page to figure out what the "current version" is.